sdcm/nemesis.py

```diff
@@ -1551,7 +1551,10 @@ def disrupt_load_and_stream(self):
                                                                                         test_data=test_data)
             for sstables_info, load_on_node in map_files_to_node:
                 SstableLoadUtils.upload_sstables(load_on_node, test_data=sstables_info, table_name="standard1")
-                SstableLoadUtils.run_load_and_stream(load_on_node)
+                # NOTE: on K8S logs may appear with a delay, so add a bigger timeout for it.
+                #       See https://github.com/scylladb/scylla-cluster-tests/issues/6314
+                kwargs = {"start_timeout": 1800, "end_timeout": 1800} if self._is_it_on_kubernetes() else {}
```

Comment:
`start_timeout =  end_timeout` - is it by intend ?

Reply:
What exactly concerns you?

The `run_load_and_stream` function is updated in this PR below to be able to set timeout for both message kinds - `start`ing one and `end`ing one.

Reply:
I wondered that it is same

Reply:
> I wondered that it is same

For non-K8S it is different, for K8S case I set it to be the same, but keeping code the more suitable way to set it to something different.